.github/workflows/release.yaml

```diff
@@ -100,6 +102,11 @@ jobs:
           POSTHOG_API_KEY: ${{ secrets.POSTHOG_API_KEY }}
           POSTHOG_ENDPOINT: ${{ secrets.POSTHOG_ENDPOINT }}
 
+      - name: Build SLSA Attestation
```

Comment:
btw, this is not taking the result of the generation and passing it to Chainloop as part of the attestation as a material. How can we do that?